nonce/nonce.go

```diff
@@ -143,20 +143,19 @@ func (ns *NonceService) encrypt(counter int64) (string, error) {
 }
 
 func (ns *NonceService) decrypt(nonce string) (int64, error) {
-	var decoded []byte
+	body := nonce
```

Comment:
Just `var body string` is fine here, and less confusing.

Reply:
How do we populate it if we aren't using a prefix?

Reply:
ohhhh now I see how I was misunderstanding. Looks good, thanks. :man_facepalming: